pandas/_typing.py

```diff
@@ -65,9 +65,19 @@
 
     from pandas.io.formats.format import EngFormatter
 
+    ScalarLike_co = Union[
+        int,
+        float,
+        complex,
+        str,
+        bytes,
+        np.generic,
+    ]
+
     # numpy compatible types
-    NumpyValueArrayLike = Union[npt._ScalarLike_co, npt.ArrayLike]
-    NumpySorter = Optional[npt._ArrayLikeInt_co]
+    NumpyValueArrayLike = Union[ScalarLike_co, npt.ArrayLike]
+    # Name "npt._ArrayLikeInt_co" is not defined  [name-defined]
```

Comment:
Looks like the same (false positive?) issue as `npt._ScalarLike_co` - just out of interest, why redefine `ScalarLike_co ` and ignore the error for `_ArrayLikeInt_co`?

Reply:
Because the scalar case was trivial while the array cases is defined using multiple private numpy typing objects

Reply:
ok clear - no objections then